libr/cons/cons.c

```diff
@@ -1268,7 +1268,7 @@ R_API int r_cons_get_column(void) {
 
 /* final entrypoint for adding stuff in the buffer screen */
 R_API int r_cons_memcat(const char *str, int len) {
-	if (len < 0) {
+	if (R_STR_ISEMPTY(str) || len < 0) {
```

Comment:
respect indentation rules  (missing space before `(`)
```suggestion
	if (R_STR_ISEMPTY (str) || len < 0) {
```